modules/user_data/main.tf

```diff
@@ -259,7 +259,7 @@ locals {
     "${path.module}/templates/tfe_vm.sh.tpl",
     {
       airgap_pathname          = local.airgap_pathname
-      airgap_url               = var.airgap_url
+      airgap_url               = var.airgap_url == null ? "" : var.airgap_url
```

Comment:
This logic can be omitted; the logic inside the template should also check for `null` rather than `""`.